go1.13/main/Dockerfile.tmpl

```diff
@@ -17,6 +17,7 @@ RUN \
         mingw-w64 \
         mingw-w64-tools \
         patch \
+        librpm-dev:i386 \
```

Comment:
@kvch I think we are missing journalbeat dependencies here. What is required for amd64 and i386 builds (I think i386 requires us to jump through some extra hoops).

Is the `main` image used for building i386 and amd64 builds? Maybe we should provide separate images. Especially the systemd dependency can be a pain.

Reply:
I guess so, but I could find a place where are defined the packages needed for any beat or the place where they are installed when you run the `mage package`

Reply:
there is a PR that create a new amd64 Docker image https://github.com/elastic/golang-crossbuild/pull/53, I will make a review of it with the changes in here

Reply:
@urso Mage installs the headers Journalbeat needs before running the default `GolangCrossBuild` target. Previously it was not added to the common Docker image to avoid installing unnecessary packages for different Beats. It is similar to Packetbeat, there the unique dependencies of the Beat are installed before crossbuild. I think it has been a good practice. I would not keep the common images as minimal as possible.

Reply:
I am not sure I follow why this PR is needed. Journalbeat installs its dependencies before running GolangCrossbuild. Once it is merged, we need to follow-up in the magefile of the Beat?

Reply:
I had a chat with @urso offline about the goal of this PR. You can find the deps of Journalbeat by platform here: https://github.com/elastic/beats/blob/master/journalbeat/magefile.go#L121-#L133